go-controller/cmd/ovnkube/ovnkube.go

```diff
@@ -544,22 +544,22 @@ func runOvnKube(ctx context.Context, runMode *ovnkubeRunMode, ovnClientset *util
 	return nil
 }
 
-type ovnkubeMasterMetrics struct {
+type leaderMetrics struct {
 	runMode *ovnkubeRunMode
 }
 
-func (m ovnkubeMasterMetrics) On(string) {
+func (m leaderMetrics) On(string) {
 	if m.runMode.ovnkubeController {
-		metrics.MetricMasterLeader.Set(1)
+		metrics.MetricOVNKubeControllerLeader.Set(1)
```

Comment:
we dont have LE for ovnkube controller right? We can remove the metrics for LE.

Reply:
hmm but in non-ic we have this right? So in default KIND deployments we still need to support this right?

Reply:
true true